scripts/mutation_test.py

```
@@ -410,67 +446,117 @@ def _run_mutants(mutants, originals):
+        sys.exit("[mutation] 白名单条目无论证，拒绝运行：%s" % ", ".join(missing))
+
+
 def main():
     ap = argparse.ArgumentParser()
     ap.add_argument("--list", action="store_true", help="只列变异点统计")
-    ap.add_argument("--max", type=int, default=0, help="最多执行 N 个变异体（冒烟）")
-    ap.add_argument("--only-op", default="", help="逗号分隔算子过滤")
+    ap.add_argument("--max", type=int, default=0, help="最多执行 N 个变异体（冒烟；不落盘报告）")
+    ap.add_argument("--only-op", default="", help="逗号分隔算子过滤（调试；不落盘报告）")
+    ap.add_argument("--min-kill-rate", type=float, default=DEFAULT_MIN_KILL_RATE,
+                    help="击杀率门禁（%%，等价体剔除后计）；低于则 exit 1（CI 用）")
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)